Pirat/Services/IConfigurationService.cs

```diff
@@ -39,6 +40,16 @@ public interface IConfigurationService
         /// </returns>
         List<string> GetRegionCodes();
 
+        void ThrowIfNotInRegion(string regionCode, Offer offer);
+
+        void ThrowIfNotConsumableCategoryInLanguage(string regionCode, string category);
+
+        void ThrowIfNotDeviceCategoryInLanguage(string regionCode, string category);
+
+        void ThrowIfNotPersonnelAreaInLanguage(string regionCode, List<string> areas);
+
+        void ThrowIfNotPersonnelQualificationInLanguage(string regionCode, List<string> qualifications);
```

Comment:
`InLanguage -> InRegion` - If you change it once, you should probably change it everywhere^^